README.md

```diff
@@ -8,6 +8,15 @@ This repository contains these plugins to support running Velero on AWS:
 
 - A volume snapshotter plugin for creating snapshots from volumes (during a backup) and volumes from snapshots (during a restore) on AWS EBS.
 
+## Compatibility
+
+Below is a listing of plugin versions and respective Velero versions that are compatible.
+
+| Provider     | Plugin Version  | Velero Version |
```

Comment:
I think we can probably drop the provider column, since this whole repo is for AWS